components/finance/FinanceDateHeader/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button } from '@material-ui/core'
+import { createStyles, makeStyles } from '@material-ui/core/styles'
+import React from 'react'
+
+import DateHelper from '../../../lib/dataHelper'
+
+function createData(date: string, value: number) {
+  return { date, value }
+}
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    todayHeader: {
+      borderRadius: 0,
+      color: 'white',
+      margin: 'auto',
+      width: '10.15rem',
+      background: '#255d6d',
+      '&:hover': {
+        background: '#255d6d',
+      },
+    },
+    dateHeader: {
+      borderRadius: 0,
+      color: 'white',
+      margin: 'auto',
+      width: '10.15rem',
+      background: '#389bb7',
+      '&:hover': {
+        background: '#389bb7',
+      },
+    },
+    values: {
+      borderRadius: 0,
+      margin: 'auto',
+      width: '10.15rem',
+      background: '#ffd2ad',
+      '&:hover': {
+        background: '#ffd2ad',
+      },
+    },
+  })
+)
+
+export default function FinanceDateHeader({ left, right }: any) {
+  const classes = useStyles()
+  const today = new Date()
+  const dates = []
+  for (let i = left; i < right; i += 1) {
+    const newDate = new Date(today.getFullYear(), today.getMonth(), today.getDate() + i)
+    dates.push(createData(newDate.toDateString(), i))
+  }
+
+  return (
+    <div className="ml-96 inline-block">
```

Comment:
use flex box (flex flow-row tailwind classes) to arrange the header dates horizontally beside each other so that it can be cutomized better tomorrow for layout

Reply:
using flex box makes it have space in between which isn't good. I have updated the classsNames. With rows decided based on space available this should be fine.